AptSpringAPI/src/main/java/com/salesforce/aptspring/Verified.java

```diff
@@ -54,26 +55,26 @@
  *  <li>@Bean annotations define at least one name for the bean.</li>
  *  <li>@Bean methods return an object (not void, not un-boxed values)</li>
  *  <li>@Bean methods are public</li>
- *  <li>@Bean methods are not static, final, native, or abstract</li>
+ *  <li>@Bean methods are not static, final, native, or abstract (support may be added for BeanFactoryPostProcessors)</li>
  *  <li>@Bean method parameters must have a @Qualifier of the bean name they expect as input or an @Value 
  *  of the property they expect spring to inject (system property or other configuration).</li>
  * </ul>
  * <p>
  * With the above restrictions in place, the following checks become feasible, and also occur at compile time:
  * </p>
  * <ul>
- *  <li>Cycles in @Configuration @Import class</li>
+ *  <li>Cycles in @Import class</li>
  *  <li>Cycles in @Bean dependency definitions</li>
```

Comment:
What about cycles in @Component?

Reply:
Hmm,  not distinguishing about cycles between beans by @Bean or @Component in line 67, though I probably should explicitly mention it.